server/sonar-web/src/main/js/api/web-api.ts

```diff
@@ -65,14 +66,16 @@ export interface Example {
   format: string;
 }
 
-export function fetchWebApi(showInternal: boolean = true): Promise<Array<Domain>> {
-  return getJSON('/api/webservices/list', { include_internals: showInternal }).then(r =>
-    r.webServices.map((domain: any) => {
-      const deprecated = !domain.actions.find((action: any) => !action.deprecatedSince);
-      const internal = !domain.actions.find((action: any) => !action.internal);
-      return { ...domain, deprecated, internal };
-    })
-  );
+export function fetchWebApi(showInternal: boolean = true): Promise<Domain[]> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type boolean trivially inferred from a boolean literal, remove type annotation ([typescript:S3257](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3257))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Fstandard_badges&issues=AWD6s19Bf6JiG_HAriM5&open=AWD6s19Bf6JiG_HAriM5)

[ISSUE_KEY]: AWD6s19Bf6JiG_HAriM5